include/mbedtls/oid.h

```diff
@@ -466,6 +466,25 @@ typedef struct mbedtls_oid_descriptor_t {
  */
 int mbedtls_oid_get_numeric_string(char *buf, size_t size, const mbedtls_asn1_buf *oid);
 
+/**
+ * \brief           Translate a string containing a numeric representation
+ *                  of an ASN.1 OID into its encoded form
+ *                  (e.g. "1.2.840.113549" into "\x2A\x86\x48\x86\xF7\x0D")
+ *                  On success, this function allocates oid->buf from the
+ *                  heap. It must be free'd by the caller.
```

Comment:
(very minor) we seem to just use "freed" (sometimes "freed with mbedtls_free()") judging by a quick grep for "must be free" :) - there is one instance of `free'd` that I could find, in x509.c